app/Http/Controllers/GridController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/*
+* Status: Public
+* Description: Grid Template
+* Default: false
+*/
+
+namespace App\Http\Controllers;
+
+use Contracts\Repositories\GridRepositoryContract;
+use Illuminate\Http\Request;
+
+class GridController extends Controller
+{
+    /**
+     * Construct the controller.
+     *
+     * @param GridRepositoryContract $promo
+     */
+    public function __construct(GridRepositoryContract $promo)
+    {
+        $this->promo = $promo;
+    }
+
+    /**
+     * Display the grid view.
+     *
+     * @param Request $request
+     * @return \Illuminate\View\View
+     */
+    public function index(Request $request)
+    {
+        // Get grid promotions
```

Comment:
You can remove this comment `// Get grid promotions`. I'm trying to reduce comments that are clearly explained by well named code